Userland/Demos/CatDog/CatDog.h

```diff
@@ -44,33 +50,14 @@ class CatDog final : public GUI::Widget {
     Gfx::IntPoint m_temp_pos;
     Core::ElapsedTimer m_timer;
 
-    int m_curr_frame = 1;
+    int m_curr_frame, m_curr_second = 0;
```

Comment:
Please don't declare multiple variables on the same line.

```suggestion
    int m_curr_frame = 0;
    int m_curr_second = 0;
```

Also, generally abbreviations should be avoided, unless they're more clear/common than the word itself (which isn't the case for "current").